check-all-vpn-exits.sh

```diff
@@ -14,13 +14,17 @@ fi
 # Name of runfile
 RUN_FILE="/run/$(basename $(readlink -f $0))"
 
+# Check for run file
```

Comment:
Das ganze Runfile-Verhalten wurde hier ziemlich umgekrempelt.

In der alten Version wurde das Runfile nicht gelöscht, wenn das Skript mit einem Fehler endete (wegen dem "if [ $? -eq 0 ]" in finish()); und dann konnte das Skript nie mehr laufen, bis das Runfile manuell gelöscht wurde.

Das neue Verhalten ist:
- wenn beim Start ein Runfile existiert, dann beende dich (das verhindert, dass das Skript zweimal gleichzeitig läuft), aber lösche nicht das existierende Runfile
- anderenfalls lege das Runfile an, und lösche es beim Beenden

Das heisst: solange ein Runfile existiert, läuft das Skript (und darf also nicht nochmal gestartet werden). Wenn kein Runfile existiert, darf das Skript gestartet werden. Ob sich das Skript mit oder ohne Fehler beendet hat, hat keinen Einfluss auf das Runfile.

Meinst du, das deckt alle Fehlersituationen ab?

Ich kann das neue Verhalten heut abend auch nochmal im Commit-Kommentar beschreiben.